packages/kernel-ui/package.json

```
@@ -0,0 +1,119 @@
+{
+  "name": "@metamask/kernel-ui",
+  "version": "0.1.0",
```

This will be bumped upon release. Our package versions are independent (i.e. don't have to be the same).
```suggestion
  "version": "0.0.0",
```